azurerm/resource_arm_container_group_test.go

```diff
@@ -0,0 +1,172 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMContainerGroup_basicLinux(t *testing.T) {
+	ri := acctest.RandInt()
+
+	config := testAccAzureRMContainerGroupBasicLinux(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMContainerGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMContainerGroupExists("azurerm_container_group.test"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAzureRMContainerGroup_basicWindows(t *testing.T) {
+	ri := acctest.RandInt()
+
+	config := testAccAzureRMContainerGroupBasicWindows(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMContainerGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMContainerGroupExists("azurerm_container_group.test"),
+				),
+			},
+		},
+	})
+}
+
+func testAccAzureRMContainerGroupBasicLinux(ri int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_container_group" "test" {
+	
+	name = "acctestcontainergroup-%d"
+	location = "%s"
```

Comment:
rather than threading this through in multiple places, can we update this to `"${azurerm_resource_group.test.location}"`?

Reply:
Done.